src/base/cservices.h

```diff
@@ -105,6 +105,8 @@ MEGAIO_IMPEXP int services_hstore_remove_handle(unsigned short type, megaHandle
 //==
 MEGAIO_IMPEXP int64_t services_get_time_ms();
 
+extern std::recursive_mutex timerMutex;
```

Comment:
Why do you move this line from `timers.hpp` to this place? Instead, I'd move the declaration from `cservices.cpp` to the place it's used: `timers.hpp`.

Reply:
This one was due to a build error under visual studio, it wouldn't link due as the definition was inside a `extern C` block, but the declaration was not.   I thought it was more natural to have the declaration and definition in the corresponding .h/.cpp files.    I've switched the declaration back to timers.hpp instead, and adjusted the definition in cservices.cpp (I guess it's there as there is no .cpp file for timers - and having the definition in the .hpp results in "multiple definition" link errors), taking it out of the `extern C` and putting it inside `karere` namespace instead.  cheers